remote-wallet/src/remote_wallet.rs

```diff
@@ -354,12 +449,47 @@ mod tests {
             }
         );
 
-        assert!(
-            RemoteWalletInfo::parse_path(format!("ledger/nano-s/{:?}/43/501/1/2", pubkey)).is_err()
+        // Other query strings get ignored
```

Comment:
It'd be better to have other query string raise an error.  It'd be especially helpful if I mistyped "key".  Capitalization should also be rejected "Key=1/2".  Duplicates too: "key=1&key=2".

Reply:
Sg, I went back and forth on that several times.